.travis.yml

```diff
@@ -0,0 +1,19 @@
+dist: jammy
+
+language: java
+jdk: openjdk17
```

Comment:
Would it be useful to test against other Java versions as well?

Reply:
This is only used to compile and not to run the sandbox, which is using it's own JetBrains JDK. The source level is set to Java 11 already to let the compiler check the sources.